seed/models/sensors.py

```diff
@@ -44,13 +45,15 @@ class Sensor(models.Model):
     column_name = models.CharField(max_length=255)
 
     class Meta:
-        unique_together = ('data_logger', 'display_name')
-        unique_together = ('data_logger', 'column_name')
+        unique_together = (
+            ('data_logger', 'display_name'),
+            ('data_logger', 'column_name'),
+        )
 
 
 class SensorReading(models.Model):
     reading = models.FloatField(null=True)
-    timestamp = models.DateTimeField()
+    timestamp = TimescaleDateTimeField(interval="1 second")
```

Comment:
This should probably stay at the default of 7 days, 1-second sized chunks are super inefficient:
https://www.timescale.com/blog/timescale-cloud-tips-testing-your-chunk-size/#finding-your-optimal-chunk-size

Reply:
is interval not the granularity of the time stored? I can't find a definition